tests/gcp/installer/util/test_data/expected_merged_group_rules.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Allow my company users and gmail users to be in my company groups.
+  group_email: my_customer
+  mode: whitelist
+  conditions:
+    - member_email: '@domain.tld'
+    - member_email: '@gmail.com'
```

Comment:
This gmail.com is merging in from the `merge_to` file.  Which I don't think is right?  If the `merge_from` file doesn't have this gmail.com, why should this be here?

Reply:
In this case replace does make more sense, I have updated the logic to replace instead of merge in this scenario.